src/vos/vos_dtx.c

```diff
@@ -685,50 +685,64 @@ dtx_rec_release(struct vos_container *cont, struct vos_dtx_act_ent *dae, bool ab
 
```

Comment:
It looks to me this dtx_rec_release() changes both pmem and dram data, if something failed in the middle,  do we need to rollback dram changes to keep pmem and dram data consistent?

Reply:
Yes, we will change on-disk layout firstly, then the DRAM changes during post_handle(). Please check the latest commit.